packages/wrangler/src/cloudchamber/build.ts

```diff
@@ -270,29 +265,30 @@ export async function pushCommand(
 }
 
 export async function ensureDiskLimits(options: {
-	requiredSize: number;
+	requiredSizeInBytes: number;
 	account: CompleteAccountCustomer;
-	containerApp: ContainerApp | undefined;
+	configDiskInBytes: number | undefined;
 }): Promise<void> {
 	const MB = 1000 * 1000;
-	const MiB = 1024 * 1024;
-	const appDiskSize = resolveAppDiskSize(options.containerApp);
 	const accountDiskSize =
 		(options.account.limits.disk_mb_per_deployment ?? 2000) * MB;
 	// if appDiskSize is defined and configured to be more than the accountDiskSize, error
-	if (appDiskSize && appDiskSize > accountDiskSize) {
+	if (
+		options.configDiskInBytes &&
+		options.configDiskInBytes > accountDiskSize
+	) {
 		throw new UserError(
-			`Exceeded account limits: Your container is configured to use a disk size of ${appDiskSize / MB} MB. However, that exceeds the account limit of ${accountDiskSize / MB}`
+			`Exceeded account limits: Your container is configured to use a disk size of ${Math.ceil(options.configDiskInBytes / MB)} MB. However, that exceeds the account limit of ${accountDiskSize / MB}MB`
```

Comment:
nit: in the first size there's a space between the number and MB, in there second there's not, I think we should be consistent in this?

basically:

```suggestion
			`Exceeded account limits: Your container is configured to use a disk size of ${Math.ceil(options.configDiskInBytes / MB)}MB. However, that exceeds the account limit of ${accountDiskSize / MB}MB`
```

or

```suggestion
			`Exceeded account limits: Your container is configured to use a disk size of ${Math.ceil(options.configDiskInBytes / MB)} MB. However, that exceeds the account limit of ${accountDiskSize / MB} MB`
```

Reply:
In the `UserError` below I can see spaces, so maybe that's the convention to follow?